arch/arm/soc/nxp_lpc/lpc54xxx/Kconfig.defconfig.lpc54114

```diff
@@ -0,0 +1,38 @@
+# Kconfig - NXP LPC54114 platform configuration options
+#
+# Copyright (c) 2017, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_LPC54114
+
+config SOC
+	string
+	default lpc54114
+
+if PINMUX
+
+config PINMUX_MCUX_LPC
+	def_bool y
+
+endif # PINMUX
+
+config GPIO
+	def_bool y
+
+if GPIO
```

Comment:
since you always enable GPIO, is the if needed? (when the 'if GPIO' goes aways this should probably also be a select in Kconfig.series